.github/workflows/pr_test.yml

```diff
@@ -1,5 +1,8 @@
 name: "PR: Run all tests"
 on:
+  push:
+    branches:
+      - 'tlim_testpr'
```

Comment:
plz add a comment that `git push origin master:tlim_testpr --force` will trigger a full PR test on the master branch.